### ISSUE TITLE
Difficult UI for adding many/all item sets to a site

### ISSUE THREAD

**Issue by zerocrates** [omeka/omeka-s#1462]
This is a close cousin of the issues with the Attachment interface from some time ago: it's pretty clunky to use when you want to do many item set assignments at once, since you have to keep using the sidebar and selecting sets one by one...

Sets also don't get removed from the sidebar when already selected, and the open section gets reset whenever one is picked.

**Comment by zerocrates** [comment]
@mebrett can you take a look over the new workflow here?

It's now on develop.

**Comment by mebrett** [comment]
What's the expected behavior?

**Comment by kimisgold** [comment]
* When selecting an item set, its parent in the right hand selector no longer collapses automatically.
  * Clicking on the owner/parent should still expand and collapse the respective item sets.
* The view scrolls a newly added item set to the top of the page.
* After an item set has been added to the resource pool, the total count and owner count of available item sets updates.
* When removing an item set from a resource pool, the row is removed from the main content area, and the item set becomes available in the right hand selector.
  * The total count and owner count should also update accordingly here.

I think that covers the main behaviors.

**Comment by mebrett** [comment]
New item sets are added at the end of the list of item sets, and the page scrolls a bit so that the item set is near the top of the page. 

Owner counts update as expected, and the owner stays open, even when other owners are open. 

The display of the delete button and the owner's email are overlapping:
<img width="1011" alt="Screen Shot 2019-11-14 at 12 58 37 PM" src="https://user-images.githubusercontent.com/2314070/68883653-7a723480-06df-11ea-9479-23feb167bfdf.png">

**Comment by kimisgold** [comment]
That looks like it's missing some of the new styles—in particular, that sortable handle column should be much narrower. Can you double check with a hard refresh?

**Comment by mebrett** [comment]
Hard refresh resolved the issue with the placement of the delete icon.

**Comment by zerocrates** [comment]
@mebrett what's your feeling on the scroll behavior? It's the same thing as happens when you add a value to the item edit form: we're trying to scroll it to the top, basically. I talked with Kim a little bit about it but didn't really come to a conclusion on whether we thought we should change it (scrolling just enough to bring it into the visible area would be the other option).

**Comment by mebrett** [comment]
I have mixed feelings about it. On the one hand, having it be near the top of the page is nice, especially since the drawer doesn't scroll down (so you're not having to move around a lot to find additional item sets). On the other hand, it means that I can't easily see everything I've already added. I know that the ones which have been added disappear from the side drawer, but if I'm trying to confirm that I've added 'everything' from a handwritten list, etc, then
